server/bub-mdt.lua

```diff
@@ -0,0 +1,15 @@
+RegisterServerEvent("mdt:server:CreateCall", function(data)    
+	exports["lb-tablet"]:AddDispatch({
+        priority = data?.isEmergency and "high" or "medium",
+        code = data?.code or '',
+        title = data?.offense or T("DEFAULT.TITLE"),
+        description = "",
+        location = {
+            label = data?.location or data?.offense or T("DEFAULT.LOCATION_LABEL"),
```

Comment:
`data.location` seems to always be empty ?
> [`server/calls.lua:L18`](https://github.com/BubbleDK/bub-mdt/blob/main/server/calls.lua#L18)

Reply:
Yes, you're right that `data?.location` is empty, but I keep it so it updates and uses the location when available. It's already linked with the UI.

Reply:
Yeah that's going to cause an issue if location is `""` the lua SCRT will consider it as valid and always set the label as an empty string, here's a quick example that I ran in fxserver:
```lua
print('^4Lua Testing^7')

local data = {
    location = "",
    offence = "manslaughter"
}

print(('Title: ^3%s^7'):format(data?.location or data?.offence or "DEFAULT"))
``` 
Output:
```bash
11:52:15 [script:lb-tablet-dis] Lua Testing
11:52:15 [script:lb-tablet-dis] Title: 
```
![image](https://github.com/user-attachments/assets/c7d9c346-0cbe-4739-8808-ae0466314b57)

Reply:
I think this works well:
```suggestion
            label = ((data?.location ~= nil and data?.location ~= "") and data?.location or data?.offense or T("DEFAULT.LOCATION_LABEL"))
```